tensorflow/contrib/rnn/python/kernel_tests/rnn_cell_test.py

```diff
@@ -42,6 +42,7 @@
 from tensorflow.python.ops import variables
 from tensorflow.python.platform import test
 from tensorflow.python.util import nest
+from tensorflow.python.training import adam
```

Comment:
Do you really need Adam, or can you get away with just calling gradients?

Reply:
you are right, it isn't required. I'll remove it.